packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolylineController.h

```diff
@@ -7,10 +7,13 @@
 
 // Defines polyline controllable by Flutter.
 @interface FLTGoogleMapPolylineController : NSObject
+@property(strong, nonatomic) GMSPolyline *polyline;
```

Comment:
This needs a documentation comment, per style guide.

Why is it public? If it's just for testing, it should be in a `GoogleMapPolylineControlller_Test.h` instead.

Reply:
Yeah, great call